action_server/src/action_server/server.py

```diff
@@ -63,6 +63,11 @@ def _add_action_cb(self, goal):
             return
 
         while not self._task_manager.done:
+            # Pass feedback to client about what type of action is running
+            feedback = action_server_msgs.msg.TaskFeedback()
+            feedback.current_subtask = self._task_manager.get_next_action_name()
```

Comment:
current = next? Wut?

Reply:
The server requests what the next action will be, send this as feedback and starts executing the action. Until the next feedback is sent or the action is finished, this is the currently active action.

Reply:
Hmm, could there be a better name for the function? It will continue to raise eyebrows like this :-)

Reply:
This is actually already on master. I should have rebased this project after merging the one that contained this commit, but cannot do that right now. I will gladly change this in another pr